packages/wallet-connection/src/agoric-wallet-connection.js

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable no-underscore-dangle */
+// @ts-check
+import { html, css, LitElement } from 'lit';
+
+import { assert, details as X } from '@agoric/assert';
+import { makeCapTP } from '@agoric/captp';
+import { Far } from '@agoric/marshal';
+import { makePromiseKit } from '@agoric/promise-kit';
+
+import 'robot3/debug';
+import { interpret } from 'robot3';
+
+import { makeConnectionMachine } from './states.js';
+import { makeAgoricIframeMessenger } from './agoric-iframe-messenger.js';
+
+// TODO: Use something on agoric.app instead.
+const DEFAULT_LOCATOR_URL =
+  'https://local.agoric.com/?append=/wallet-bridge.html';
+
+export const makeAgoricWalletConnection = (
```

Comment:
Sorry about the diff. The only thing I changed in this file was this top part.

Reply:
If the diff happens in the same file, that's fine... I just turn off the whitespace change detection:

![image](https://user-images.githubusercontent.com/457244/135377827-1a26c045-ebcb-42e2-af0e-10f7c6def02c.png)